docs/admin-sdk/docs/guide/2_api-reference/in-app-purchases/in-app-purchases.md

```diff
@@ -0,0 +1,15 @@
+# In-App Purchase Flow
```

Comment:
Looks good. One small thing I would add is the first Shopware Version where this feature is available at the top, see https://developer.shopware.com/resources/admin-extension-sdk/api-reference/cms/registerCmsBlock.html as an example

Reply:
Maybe we can add this later, since this is only going into a feature-branch? Right now, we have no clue, what the release minor will be, still a lot of dependency on the SBP.

Reply:
I'm adding it anyway as 6.7.0, just so we have it. We can always change it.